src/components/pages/Office/OfficeInfo/OfficeInfo.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC } from 'react';
+import { useRouter } from 'next/router';
+import useSWR from 'swr';
+
+import { fetcher } from '@/constants';
+import { Loader } from '@/components/shared/Loader';
+
+export const OfficeInfo: FC = () => {
+  const {
+    query: { officeInfoId },
+  } = useRouter();
+
+  const { data, isLoading } = useSWR(officeInfoId ? `/api/officeInfo/${officeInfoId}` : null, fetcher);
```

Comment:
Need more variable understanding of naming, here you have office info, `data` it's bad naming, e.g.:
```suggestion
  const { data: officeInfo, isLoading } = useSWR(officeInfoId ? `/api/officeInfo/${officeInfoId}` : null, fetcher);
```